ern-orchestrator/src/codepush.ts

```diff
@@ -99,6 +100,7 @@ export async function performCodePushPromote(
     rollout?: number
     reuseReleaseBinaryVersion?: boolean
     targetBinaryVersion?: string
+    skipNativeDependenciesVersionAlignedCheck?: boolean
```

Comment:
Minor: Looks like the other entries are (roughly) in alphanumeric order. Want to move this line up above `targetBinaryVersion`? Same on line 93.

Reply:
Yeah, most are aligned, not all. I'll add a commit after re arranging.